### PR TITLE
Cleanup www.mobifyjs.com

### PR DESCRIPTION
**Pull request by johnboxall**
- Add <meta> attributes
- Remove unused JavaScript
- Delete unused files
- Update video links to the correct place

### CODE REVIEW

File: www/docs/index.md [hunk 1]
```diff
@@ -38,8 +42,8 @@ Insert the Mobify.js tag **immediately** after the opening _<head>_ tag on the w
     document.write('<plaintext style="display:none">');
 
     setTimeout(function() {
-        var mobifyjs = document.createElement('script')
-          , script = document.getElementsByTagName('script')[0];
+        var mobifyjs = document.createElement('script');
+        var script = document.getElementsByTagName('script')[0];
```
Comment (jansepar) [inline www/docs/index.md:46]: Are we changing this convention?

Comment (johnboxall) [inline www/docs/index.md:46]: Yup.

Comment (jansepar) [inline www/docs/index.md:46]: Why?

Comment (johnboxall) [inline www/docs/index.md:46]: I find this format, though more verbose, lets you make less decisions per line. Declaring a variable? Use `var x;`. Always. This way you never have to think about whether you should put a semicolon on the end of the line, how you should indent, whether you should use a `var` etc...

Comment (jansepar) [inline www/docs/index.md:46]: Agreed. I'm happy with this change in convention.

**Pull request merged**